Aptoide/appcoins-wallet-android
# [APPC-1137] Handle item already owned

**What does this PR do?**

   Handles the cases where a transaction is completed but the purchased is not consumed by showing a dialog and returning the purchase again once the user dismisses the dialog, so that the application can try and consume it again

**Note**: Don’t accept this PR yet, as it doesn’t have the string to be shown on the dialog.

**Where should the reviewer start?**

PaymentMethodsFragment.java

**How should this be manually tested?**

Open Trivial Drive project and pull the last changes from dev. Then comment line 797 that should be the consumeAsyncInternal method. 
Buy gas in any payment method, the purchase should not be consumed (your shall not receive gas, due to commented line), now try and buy gas again and the error dialog should appear. Once you dismiss it, the purchase should be consumed and the tank should be filled.
 
**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1137

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
